formats/json-refract-schema.json

```diff
@@ -0,0 +1,180 @@
+{
+  "oneOf": [
+    { "$ref": "#/definitions/element" }
+  ],
+  "definitions": {
+    "element": {
+      "title": "Element",
+      "type": "object",
+      "properties": {
+        "element": {
+          "type": "string"
+        },
+        "meta": {
+          "type": "object",
+          "properties": {
+            "id": { "$ref": "#/definitions/element" },
+            "title": { "$ref": "#/definitions/string-element" },
+            "description": { "$ref": "#/definitions/string-element" },
+            "ref": { "$ref": "#/definitions/element-pointer" },
+            "classes": { "$ref": "#/definitions/array-string-element" },
+            "links": { "$ref": "#/definitions/array-link-element" }
+          },
+          "additionalProperties": false
+        },
+        "attributes": {
+          "type": "object",
+          "patternProperties": {
+            "": {
+              "$ref": "#/definitions/element"
+            }
+          }
+        },
+        "content": {
+          "oneOf": [
+            {
+              "type": [
+                "string",
+                "number",
+                "boolean",
+                "null",
+                "array"
+              ],
+              "items": {
+                "$ref": "#/definitions/element"
+              }
+            },
+            {
+              "$ref": "#/definitions/element"
+            },
+            {
+              "$ref": "#/definitions/key-value-pair"
+            }
+          ]
+        }
+      },
+      "required": [
+        "element"
+      ],
+      "additionalProperties": false
+    },
+    "key-value-pair": {
+      "title": "Key Value Pair",
+      "type": "object",
+      "properties": {
+        "key": {
+          "$ref": "#/definitions/element"
+        },
+        "value": {
+          "$ref": "#/definitions/element"
+        }
+      },
+      "required": ["key"],
+      "additionalProperties": false
+    },
+    "string-element": {
+      "title": "String Element",
+      "allOf": [
+        { "$ref": "#/definitions/element" },
+        {
+          "properties": {
+            "element": {
+              "enum": ["string"]
```

Comment:
Element being a string is inherited from another `allOf`. `{ "$ref": "#/definitions/element" }`.

This would also be necessary as `enum` already implies the type as no other types are allowed in the enum value.

+ this is within an `allOf` which requires `{ "$ref": "#/definitions/element" },` to pass which does have the type configur